www/test/functional/page-objects/report.js

```diff
@@ -21,19 +21,27 @@
   };
 
   this.fillRisk = function (risk) {
-    return element(by.cssContainingText('option', risk)).click();
+    element(by.cssContainingText('option', risk)).click();
```

Comment:
I just changed this return to allow call this method in a nested way from the test.